packages/clippy/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "@react95/clippy",
+  "version": "0.0.1",
+  "description": "Our old good friend Clippy as a React Hook!",
+  "keywords": [
+    "React95",
+    "custom",
+    "hook",
+    "clippy"
+  ],
+  "author": "ggdaltoso <ggdaltoso@gmail.com>",
+  "homepage": "https://react95.github.io/React95",
+  "license": "MIT",
+  "main": "index.js",
+  "publishConfig": {
+    "access": "public",
+    "directory": "dist"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/React95/React95.git"
+  },
+  "scripts": {
+    "prebuild": "rimraf ./dist",
+    "build": "cross-env NODE_ENV=production babel ./src --out-dir dist --copy-files",
+    "prepublish": "yarn build",
+    "prepublishOnly": "node ../../scripts/prepublish.js"
+  },
+  "bugs": {
+    "url": "https://github.com/React95/React95/issues"
+  },
+  "devDependencies": {
+    "@testing-library/react": "^10.0.5",
+    "@testing-library/react-hooks": "^3.2.1",
+    "rimraf": "^3.0.2"
+  },
+  "dependencies": {
+    "clippyjs": "^0.0.3",
+    "jquery": "^3.5.1",
+    "react": "^16.13.1"
```

I think that this dependency should be at `peerDependencies`, what do you think?
```suggestion
"peerDependencies": {
  "react": ">= 16.8.0",
}
```